cluster/examples/kubernetes/ceph/crds.yaml

```diff
@@ -1298,7 +1298,6 @@ spec:
                                   format: int32
                                   type: integer
                               required:
-                                - preference
```

Comment:
The crds.yaml is now generated automatically, so you'll need to change the generation attributes in [types.go](https://github.com/rook/rook/blob/master/pkg/apis/ceph.rook.io/v1/types.go). However, I'm not sure we'll have much control over removing these `preference` requirements since we don't own those types.

Reply:
If we can't remove the `preference` requirements in the CRDs, should we then change the example files so that they include the `preference` attribute where it's missing?